spyder/plugins/ipythonconsole/widgets/namespacebrowser.py

```diff
@@ -39,12 +39,20 @@ def get_value(self, name):
         reason_big = _("The variable is too big to be retrieved")
         reason_not_picklable = _("The variable is not picklable")
         reason_dead = _("The kernel is dead")
-        reason_other = _("An unkown error occurred. Check the console because "
-                         "its contents could have been printed there")
+        reason_other = _("An unkown error occurred. Check the console because"
+                         " its contents could have been printed there")
         reason_comm = _("The comm channel is not working")
+        reason_missing_package = _("The required package '{module}' to open "
+                                   "this variable is not installed")
```

Comment:
```suggestion
        reason_missing_package_installer = _(
            "The '{}' package is required to open this variable. "
            "Unfortunately, it's not part of our installer, which means your "
            "variable can't be displayed by Spyder."
        )
```

- Improve message and variable name.
- Don't use a variable for string replacement to avoid translators to wrongly translate it.